packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -24,6 +24,7 @@ import { Link } from '@bbc/psammead-story-promo';
 import { oneOf, shape, string } from 'prop-types';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
 import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { formatDuration } from '@bbc/psammead-timestamp-container/utilities';
```

Comment:
Not quite sure how `formatDuration` will work since it is only published in version 2.7.3., but Psammead uses version 2.7.2
It seems like in cases like this we usually create 2 separate PRs, where we would firstly publish functionality for `timestamp-container`, and in another PR we would pull in the new version of `timestamp-container` and use `formatDuration` in radio-schedules.
Is it going to work fine in one PR? Asking because for me locally tests are complaining about using `formatDuration` function

Reply:
Thanks Olga. The formatDuration work has been done in another PR and then pulled in here.